src/structures/vroom/raw_route.cpp

```diff
@@ -69,7 +69,7 @@ void RawRoute::update_amounts(const Input& input) {
       current_nb_pickups += 1;
       break;
     case JOB_TYPE::DELIVERY:
-      assert(job.delivery <= current_pd_load);
```

Comment:
Makes sure we don't end up in a situation where we deliver more than what's been picked along the route. You should be able to keep this.